coramin/domain_reduction/tests/test_decomp.py

```diff
@@ -0,0 +1,675 @@
+from coramin.domain_reduction.dbt import TreeBlock, TreeBlockError, convert_pyomo_model_to_bipartite_graph, \
```

Comment:
This is minor and should not hold up the PR, but can we change the name of this file to test_dbt? "decomp" is really general and can refer to many things. dbt is the shortened named for decomposed_bounds_tightening.

Reply:
Agreed.